frontend/src/components/Client/ReviewsCarousel/ReviewsCarousel.js

```diff
@@ -0,0 +1,26 @@
+import AliceCarousel from 'react-alice-carousel'
+import "react-alice-carousel/lib/alice-carousel.css"
+
+const ReviewsCarousel = ({ content }) => {
+
+  const responsive = {
+    0: { items: 1 },
+    640: { items: 2 },
+    1024: { items: 3, itemsFit: 'contain' },
```

Comment:
if possible, you should add those breakpoints to the `helper.js` plug those values in as keys instead of hardcoding the numbers here